spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,62 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    let(:test_ot_1) { FactoryGirl.create(:orchestration_template_cfn) }
+    let(:test_content_1) { "{ 'Description': 'Test content 1' }\n" }
+
+    it 'copy template with the same content' do
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      run_post(orchestration_templates_url(test_ot_1.id), gen_request(:copy))
+      expect(response).to have_http_status(:bad_request)
+    end
+
+    it 'copy single template with the different content' do
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      expected = {
+        'content' => test_content_1
+      }
+
+      expect do
+        run_post(
+          orchestration_templates_url(test_ot_1.id),
+          gen_request(:copy, :content => test_content_1)
+        )
+      end.to change(OrchestrationTemplateCfn, :count).by(2)
```

Comment:
Why did it change by 2? Is it because the execution is where the lazy-evaluated `test_ot_1` gets created? For this reason I think it's better to avoid `let`s and create everything you are going to test in the setup phase in the body of the test. Duplication is OK in tests and can improve readability